src/index.js

```diff
@@ -42,4 +20,44 @@ function generateResult() {
   });
 }
 
-generateResult();
+const sendScores = async () => {
+  const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/uwJ01ZLnBAIiUKyNJXTe/scores/';
+  const yourName = document.getElementById('yourName');
+  const user = yourName.value;
+  const yourScore = document.getElementById('yourScore');
+  const score = yourScore.value;
+  const response = await (await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+  })).json();
+  yourName.value = '';
+  yourScore.value = '';
+  return response;
+};
+
```

Comment:
**Changes Required**
- [ ] The function `sendScores` is accessing the API end point and manipulating the `DOM` at the same time. This makes this function very difficult to test and less readable. Let us try to refactor it so that it only focuses on posting data to the endpoint.

Reply:
Well done